src/components/SideInformationsContainer/SideInformationsContainer.tsx

```diff
@@ -0,0 +1,19 @@
+import { ExplanationCard } from "./ExplanationCard";
```

Comment:
What do you think the name of this component be `CardContent`? @valdersonjr @enzoggqs

Reply:
I think it would be better than the current one.